pagopa/io-app
# feat: [IOCOM-875] SEND Timeline with new DS

⚠️ This PR depends on #5700  ⚠️

## Short description
This PR adds the SEND timeline to the SEND message details, with the new DS.

|Simple timeline|Long timeline|
|-|-|
|![Simulator Screenshot - iPhone 15 - 2024-04-18 at 17 27 17](https://github.com/pagopa/io-app/assets/5150343/fb9ae0b3-ee03-4357-9f55-a37c2ed98523)|![Simulator Screenshot - iPhone 15 - 2024-04-18 at 17 24 22](https://github.com/pagopa/io-app/assets/5150343/427ccbaa-d8a9-49a8-855c-7ee887b107dd)|

## List of changes proposed in this pull request
- `TimelineListItem` component to handle both menu entry and bottom sheet 

## How to test
Using the io-dev-api-server, generate some SEND messages and check that the timeline works as expected.
